data/org.freedesktop.UDisks2.xml

```diff
@@ -1800,10 +1800,14 @@
         ChangePassphrase:
         @passphrase: The existing passphrase.
         @new_passphrase: The new passphrase to use.
-        @options: Options (currently unused except for <link linkend="udisks-std-options">standard options</link>).
+        @options: Options - known options (in addition to <link linkend="udisks-std-options">standard options</link>) includes <parameter>old_keyfile_contents</parameter> and <parameter>old_keyfile_contents</parameter> (of type 's' or 'ay').
 
         Changes the passphrase to @new_passphrase. An existing passphrase is required.
 
+        If <parameter>old_keyfile_contents</parameter> or
+        <parameter>old_keyfile_contents</parameter> are given, they take
```

Comment:
``old_keyfile_contents`` mention twice here. And the same applies to the ``@options...`` line above.